src/Tinkoff.ISA.API/Startup.cs

```diff
@@ -60,6 +86,22 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env, ILoggerF
             }
 
             app.UseMvc();
+            
+            recurringJobManager.AddOrUpdate<JiraJob>(
```

Comment:
I had to add this reference to have an ability to configure new providers in main ISA project. 
Further this code should be in Tinkoff.ISA.Jira, Tinkoff.ISA.Confluence and so on...